scout/server/blueprints/diagnoses/controllers.py

```diff
@@ -25,7 +25,22 @@ def disease_entry(store, disease_id) -> dict:
 
 
 def disease_terms(store: MongoAdapter, query: str, source: str) -> dict:
-    """Retrieve disease terms, queried by source or a text-match for the disease description."""
+    """Retrieve disease terms, queried by source or a text-match for the disease description.
+    disease_term[genes] are populated with both hgnc_id and symbol"""
+
+    disease_data = list(store.query_disease(query=query, source=source, filter_project=None))
+
+    for disease in disease_data:
+        gene_ids = disease.get("genes", [])
+        gene_symbols = []
```

Comment:
It's not going to be symbols only but dictionaries with hgnc ids and symbols. Otherwise it's confusing I think